base/file.jl

```diff
@@ -115,7 +115,18 @@ function cp(src::AbstractString, dst::AbstractString; remove_destination::Bool=f
         FS.sendfile(src, dst)
     end
 end
-mv(src::AbstractString, dst::AbstractString) = FS.rename(src, dst)
+
+function mv(src::AbstractString, dst::AbstractString; remove_destination::Bool=false)
+    if ispath(dst)
+        if remove_destination
+            rm(dst; recursive=remove_destination)
```

Comment:
Should this just be `recursive=true` for clarity's sake? It looks like a typo if you glance at it too quickly.